src/controllers/user.js

```diff
@@ -174,3 +174,54 @@ export const deleteNote = async (req, res) => {
     return sendDataResponse(res, 400, { err: err.message })
   }
 }
+
+export const getAllNotes = async (req, res) => {
+  const userId = Number(req.params.id)
+  const whereData = {}
+  whereData.studentId = userId
+
+  try {
+    const foundNotes = await Note.findAll({ whereData })
+
+    if (foundNotes.length === 0) {
```

Comment:
Remove this conditional statement. This is not a valid error case - returning an empty list of notes is expected. Let the client side handle what to show the user if there are no notes.